pkg/webhook/internal/metrics/metrics.go

```diff
@@ -30,11 +30,32 @@ var (
 			Name: "controller_runtime_webhook_latency_seconds",
 			Help: "Histogram of the latency of processing admission requests",
 		},
-		[]string{"webhook"},
+		[]string{"webhook", "code"},
```

Comment:
Hi @directxman12 Maybe you can advise if this is the way to go since I add this because of [pkg/webhook/server.go:131](https://github.com/kubernetes-sigs/controller-runtime/pull/944/files#diff-fbc18bb07cdd05391b7081acc1dfe170L131) but from my point of view this is not optimal since it multiplies the number of metrics by status code.

Reply:
Removed this in https://github.com/kubernetes-sigs/controller-runtime/pull/944/commits/374972fdbad05f4b8afa945d13aee1674dfa70c4 because it caused a BC break in one of my dashboards